.circleci/config.yml

```diff
@@ -77,14 +77,14 @@ jobs:
       - run:
           name: 🏗️ build dash
           command: |
-            . venv/bin/activate && pip install --upgrade -e . --quiet && mkdir packages
+            . venv/bin/activate && mkdir packages
             # build main dash
-            git clone --depth 1 https://github.com/plotly/dash.git dash-main
-            cd dash-main && pip install -e .[ci,testing] && python setup.py sdist && mv dist/* ../packages/
-            cd dash-renderer && npm i --ignore-scripts && npm run build
+            git clone --branch assets-#597 --depth 1 https://github.com/plotly/dash.git dash-main
```

Comment:
`git clone --branch assets-#597`
Should this refer to a specific branch? Was it here for testing and you were going to change it before merging? Seems like it would be out-of-date rather quickly...just double-checking.

Reply:
this is because we are doing two PR in parallel, and will get rid of this once the dash one get merged